### PR TITLE
get breakpoints shots implementation into storybok/pages/ladle modes

### PR DESCRIPTION
**Pull request by d-ivashchuk**

### CODE REVIEW

File: examples/example-ladle/build/assets/axe.2fd54cbc.js
Comment (chriskalmar) [inline examples/example-ladle/build/assets/axe.2fd54cbc.js:1]: This looks like a build asset.

File: src/shots/shots.ts
Comment (chriskalmar) [inline src/shots/shots.ts:1]: Could we include the breakpoint (if provided) in this log?
```
`Taking screenshot of '${shotItem.shotName}'`,
```

**Review by chriskalmar — CHANGES_REQUESTED** [review]

**Comment by b3nThomas** [comment]
Hello there. Apologies if me commenting here is inappropriate, but any updates on this PR?
This feature would be very positively welcomed 🙂

**Comment by d-ivashchuk** [comment]
You can already use the feature in an experimental build 😀

```js
module.exports = {
  storybookShots: {
    storybookUrl: './storybook-static',
    breakpoints: [640, 1024],
  },
  lostPixelProjectId: 'xllx',
  apiKey: process.env.LOST_PIXEL_API_KEY,
};
```

`uses: lost-pixel/lost-pixel@feature/breakpoint-shots`

So far we are not merging it and gathering the feedback!

**Pull request merged**